Enhances `planner plan add`, Closes #4400

nicodecleyre opened the pull request:
This PR enhances `planner plan add`.

Closes #4400

milanholemans commented:
Nicely done, will review it ASAP!

Adam-it commented on docs/docs/cmd/planner/plan/plan-add.md:
```diff
@@ -29,23 +32,25 @@ m365 planner plan add [options]
 ## Examples
 
-Adds a Microsoft Planner plan with the name _My Planner Plan_ for Group _233e43d0-dc6a-482e-9b4e-0de7a7bce9b4_
+Adds a Microsoft Planner plan with a Group by id
```
we could add an example of adding a plan to a roster

Adam-it reviewed (approved):
@nicodecleyre looks and works really good 👍
you rock 🤩
![image](https://user-images.githubusercontent.com/58668583/216337583-d926b377-ec11-4eab-85a0-07aa254696fd.png)

it was the first time for me to play a bit with roster plans 😅
I added one small comment to provide and example for the `rosterId` option but I guess it is not a must have 😉 If you will find some time please do and if not I will do it when merging 👍

Adam-it commented:
ready to merge 🚀

nicodecleyre commented:
> I added one small comment to provide and example for the `rosterId` option but I guess it is not a must have 😉 If you will find some time please do and if not I will do it when merging 👍

For me it was the first time too to play with roster plans. that's one of the nice things by contributing to the CLI for Microsoft 365, you learn to work with things that you might not otherwise (immediately) come into contact with and that way you create nice insights!

Added the example! Thank you for your review, you are awesome! 🤩

Adam-it commented:
merged manually
